Dockerfile.ci

```diff
@@ -1,5 +1,8 @@
 # /bin/terraform
-FROM hashicorp/terraform:0.12.8 AS terraform-binary
+FROM hashicorp/terraform:0.12.11 AS terraform-binary
+
+# /bin/jsonnet
+FROM bitnami/jsonnet:0.14.0 as jsonnet-binary
```

Comment:
jsonnet was accidentally removed from CI image on 213c3c0b66

the previous image `sparkprime/jsonnet` hasn't been updated in a long time. Bitnami image seems to be fairly up to date.